### PR TITLE
Update Home Assistant icon and color

### PR DESCRIPTION
**Pull request by jwr1**
homeassistant

**Similarweb rank:** [9,456](https://www.similarweb.com/website/home-assistant.io/)

### Checklist
  - [x] I updated the JSON data in `_data/simple-icons.json`
  - [x] I optimized the icon with SVGO or SVGOMG
  - [x] The SVG `viewbox` is `0 0 24 24`

### Description
See this blog for the new logo's details: [A refreshed logo for Home Assistant!](https://www.home-assistant.io/blog/2023/09/17/a-refreshed-logo-for-home-assistant/)

The new icon and color were grabbed from here: https://github.com/home-assistant/assets/blob/0f69676da79c3881e7dfca6d6c0a5739f88273d8/logo/home-assistant-logo.zip

### CODE REVIEW

File: _data/simple-icons.json [hunk 1]
```diff
@@ -5951,8 +5951,8 @@
         },
         {
             "title": "Home Assistant",
-            "hex": "41BDF5",
-            "source": "https://github.com/home-assistant/assets/blob/1e19f0dca208f0876b274c68345fcf989de7377a/logo/logo-small.png",
+            "hex": "18BCF2",
+            "source": "https://github.com/home-assistant/assets/blob/0f69676da79c3881e7dfca6d6c0a5739f88273d8/logo/home-assistant-logo.zip",
```
Comment (service-paradis) [inline _data/simple-icons.json:5955]: We can also add https://design.home-assistant.io/#brand/logo as guidelines.

File: _data/simple-icons.json [hunk 2]
```diff
@@ -5951,8 +5951,8 @@
...
             "license": {
                 "type": "CC-BY-NC-SA-4.0"
             }
```
Comment (service-paradis) [inline _data/simple-icons.json:5958]: As mentioned in https://github.com/home-assistant/assets/tree/0f69676da79c3881e7dfca6d6c0a5739f88273d8/logo#copyright-notice, we should remove the license.

> Please note that this logo is not released under the CC license. All rights reserved.

File: icons/homeassistant.svg [hunk 3]
```diff
@@ -1 +1 @@
...
\ No newline at end of file
+<svg role="img" viewBox="0 0 24 24" xmlns="http://www.w3.org/2000/svg"><title>Home Assistant</title><path d="M22.94 10.63 13.06.75a1.5 1.5 0 0 0-2.12 0l-9.88 9.88A4.18 4.18 0 0 0 0 13.19v9c0 .82.68 1.5 1.5 1.5h9.23l-4.07-4.07c-.2.08-.43.12-.66.12a2.05 2.05 0 1 1 1.94-1.39l3.16 3.17V9.92a2.05 2.05 0 1 1 1.8 0v8.13l3.15-3.14a2.05 2.05 0 1 1 1.24 1.3l-4.39 4.4v3.08h9.6c.82 0 1.5-.68 1.5-1.5v-9c0-.83-.48-1.98-1.06-2.56z"/></svg>
```
Comment (service-paradis) [inline icons/homeassistant.svg:1]: I see a small difference in the middle circle. Can you take another look?
image

**Review by service-paradis — CHANGES_REQUESTED** [review]
Thank you @jwr1 for your contribution! I left some comments before we can merge it.

File: icons/homeassistant.svg (hunk 3 above)
Comment (jwr1) [reply icons/homeassistant.svg]: This should be good now. I previously used a precision of 2; now it's 3.

**Review by service-paradis — APPROVED** [review]
LGTM 🎉 thank you @jwr1!

**Pull request merged**